homotopy-web/src/app/diagram_gl/renderer/scene.rs

```diff
@@ -117,11 +126,11 @@ impl Scene {
         sphere_mesh.mk_point(p);
         sphere_mesh.inflate_point_3d(p, geometry_samples, &VertexShape::Circle);
         if let Some(sphere_buffers) = sphere_mesh.buffer_tris(ctx)?.into_iter().next() {
-            self.sphere = Some(vertex_array!(
+            self.sphere = Some(Rc::new(vertex_array!(
```

Comment:
Vertex arrays are reference counted under the hood so you don't need to add an `Rc` here, just `clone` the array

Reply:
Actually that's total nonsense ignore me this is correct